### PR TITLE
Test gh action

### PR DESCRIPTION
**Pull request by sergiogp98**
Hi!

This PR add some test:

- [x] Manager and agent connection
- [x] Check agent _ossec.log_ is reporting to manager

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -1,50 +1,56 @@
...
 
-Kitchen-vagrant
-===============
+## Kitchen-vagrant
 
-## Prerequisites
+### Prerequisites
 - Vagrant
 - Virtualbox
+- Depependancies:
```
Comment (Zenidd) [inline TESTING.md]: Check this typo, it should be `dependencies`

File: TESTING.md [hunk 2]
```diff
@@ -1,50 +1,56 @@
...
+``
 
-Kitchen dokken
-==============
+## Kitchen dokken
 
-## Prerequisites
+### Prerequisites
 - Docker
+- Depependancies:
```
Comment (Zenidd) [inline TESTING.md]: Same as above, please, fix this typo

File: TESTING.md [hunk 3]
```diff
@@ -1,50 +1,56 @@
...
+bundle config set --local without 'vagrant'
+bundle install
+``
 
-Kitchen command
-===============
+## How to use?
 
-Kitchen commands To create an environment with Wazuh Chef cookbooks:
+Once you select the driver and install all the dependancies, you can create your first node by running the following Kitchen CI commands:
```
Comment (Zenidd) [inline TESTING.md]: Could you also replace `dependancies` with `dependencies`?

**Review by xr09 — APPROVED** [review]

**Review by Zenidd — APPROVED** [review]
Really nice work @sergiogp98, LGTM!

**Review by xr09 — APPROVED** [review]
LGTM!

**Pull request merged**